aws/build_lambda_code.py

```diff
@@ -33,18 +34,23 @@
 client = MongoClient(conn_uri)
 db = client[db_name]
 fs = GridFS(db)
+headers = {{
+    'Access-Control-Allow-Origin': '*',
```

Comment:
can we change this from a wild card back to `playimaginate.com` specifically?
A little overkill maybe, but good security practice

Reply:
Sure, I will do that locally just so we can keep track of the updated index.py as well